flink-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/TableInputFormat.java

```diff
@@ -41,7 +41,7 @@
 /**
  * {@link InputFormat} subclass that wraps the access for HTables.
  */
-public abstract class TableInputFormat<T extends Tuple> extends RichInputFormat<T, TableInputSplit> {
+public abstract class TableInputFormat<T> extends RichInputFormat<T, TableInputSplit> {
```

Comment:
I'd rather rename this class to `AbstractTableInputFormat<T>` and add a new class `TableInputFormat<T extends Tuple>` to keep the API stable.